pom.xml

```diff
@@ -483,7 +491,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
-            <version>3.3.0</version>
+            <version>4.0.0</version>
```

Comment:
I think we can upgrade these packages as part of a separate PR to ensure that this does not break anything in the other implementations.